src/lib/__tests__/total.test.js

```diff
@@ -5,10 +5,11 @@ describe("total", () => {
     .fn()
     .mockReturnValue(Promise.resolve({ headers: { "x-total-count": "50" } }))
   it("loads the path and passes in the token", () =>
-    totalViaLoader(loader, {}, { size: 10, extra_option: 1 }).then(total => {
-      expect(loader.mock.calls[0][0].size).toBe(0)
-      expect(loader.mock.calls[0][0].total_count).toBe(true)
-      expect(loader.mock.calls[0][0].extra_option).toBe(1)
-      expect(total).toBe(50)
-    }))
+    totalViaLoader(loader, {}, { size: 10, extra_option: 1 })
+      .then(total => {
+        expect(loader.mock.calls[0][0].size).toBe(0)
+        expect(loader.mock.calls[0][0].total_count).toBe(true)
+        expect(loader.mock.calls[0][0].extra_option).toBe(1)
+        expect(total).toBe(50)
+      }))
```

Comment:
Just confirming that Prettier is installed in your editor, and you have it set to format on save?

Reply:
I'm using the default .vscode config files per project, I haven't set up prettier independently. Prettier does run on a pre-commit hook though, so it should be a non-issue?

Reply:
When booting VSCode it should read the recommended plugin list and if you install them then the default `.vscode` settings will be picked up in your editor, including fix on save (eslint) and format on save (prettier).

Reply:
I think prettier is at war with ESLint in spite of my attempts to reconcile them. Yesterday, running `yarn prettier-project` worked beautifully with no conflicts. Today when I run it it generates 800 new lint issues. I'm not sure what's happening.

Reply:
If you ping @orta he will help get your environment in sync.